src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -37,58 +34,30 @@ const GovernanceSection = () => {
       <AccordionDetails sx={({ spacing }) => ({ padding: `0 ${spacing(3)}` })}>
         {claimingApp || fetchingSafeClaimingApp ? (
           <WidgetBody>
-            <Grid spacing={3} container>
-              <Grid item xs={12} md={6} lg={8}>
-                <Card sx={{ height: '300px' }}>
-                  {claimingApp ? (
-                    <SafeAppsErrorBoundary render={() => <SafeAppsLoadError onBackToApps={() => {}} />}>
-                      <AppFrame
-                        appUrl={`${claimingApp.url}${SNAPSHOT_WIDGET_ID}`}
-                        allowedFeaturesList={getAllowedFeaturesList(claimingApp.url)}
-                        isQueueBarDisabled
-                      />
-                    </SafeAppsErrorBoundary>
-                  ) : (
-                    <Box
-                      sx={{ height: '300px' }}
-                      display="flex"
-                      alignItems="center"
-                      justifyContent="center"
-                      textAlign="center"
-                    >
-                      <Typography variant="h1" color="text.secondary">
-                        Loading Snapshot...
-                      </Typography>
-                    </Box>
-                  )}
-                </Card>
-              </Grid>
-              <Grid item xs={12} md={6} lg={4}>
-                <Card sx={{ height: '300px' }}>
-                  {claimingApp ? (
-                    <SafeAppsErrorBoundary render={() => <SafeAppsLoadError onBackToApps={() => {}} />}>
-                      <AppFrame
-                        appUrl={`${claimingApp.url}${CLAIMING_WIDGET_ID}`}
-                        allowedFeaturesList={getAllowedFeaturesList(claimingApp.url)}
-                        isQueueBarDisabled
-                      />
-                    </SafeAppsErrorBoundary>
-                  ) : (
-                    <Box
-                      sx={{ height: '300px' }}
-                      display="flex"
-                      alignItems="center"
-                      justifyContent="center"
-                      textAlign="center"
-                    >
-                      <Typography variant="h1" color="text.secondary">
-                        Loading Claiming app...
-                      </Typography>
-                    </Box>
-                  )}
-                </Card>
-              </Grid>
-            </Grid>
+            <Card className={css.widgetWrapper}>
+              {claimingApp ? (
+                <SafeAppsErrorBoundary render={() => <SafeAppsLoadError onBackToApps={() => {}} />}>
+                  <AppFrame
+                    // appUrl={`${claimingApp.url}#widget`}
+                    appUrl="http://localhost:3001/safe-claiming-app#widget"
```

Comment:
👀

Reply:
To be replaced once [the](https://github.com/safe-global/safe-react-apps/pull/582) is merged

Reply:
I've updated the PR description with this info